openpype/hosts/maya/api/lib.py

```diff
@@ -1751,18 +1751,24 @@ def remove_other_uv_sets(mesh):
             cmds.removeMultiInstance(attr, b=True)
 
 
-def get_id_from_history(node):
+def get_id_from_sibling(node, history_only=True):
     """Return first node id in the history chain that matches this node.
 
     The nodes in history must be of the exact same node type and must be
     parented under the same parent.
 
+    If no matching node is found in history, the siblings of the node
+    are checked. Additionally to having the same parent, the sibling must
+    be marked as 'intermediate object'.
```

Comment:
The description here and also in the *Args* description down below this feels incorrect. It **always** checks the siblings but only takes those that are also in the history. Right?

Reply:
I see what you mean, and you are right for the "Args" description.

However, for the main description, I think it is correct:
```
If no matching node is found in history, the siblings of the node are checked.
...
```
The interesting nodes in the history are indeed also siblings, but in the case none is found, the process will look in the siblings regardless of the history.

The way you say it is correct when considering the end result: The siblings are always checked, and we only takes those also in the history (if found). (else we look at all the siblings)
And I think the way I describe it is closer to the process itself: If not found in history (from the siblings), the process looks at (all) the siblings.

But I see your point, and I should probably rewrite it.
(I have never been good at documentation/comments)